src/main/java/com/easyrest/facade/admin/ActionOnClientFacade.java

```diff
@@ -0,0 +1,46 @@
+package com.easyrest.facade.admin;
+
+import com.easyrest.components.admin.ActionOnClient;
+import org.openqa.selenium.WebDriver;
+
+/**
+ * This class is facade of ActionOnClient component that contains methods to make action on client - ban client
+ * or make him active and checks if client moved to correct list.
+ * Client - any type of user role (owner or administrator or admin or waiter or moderator or user).
+ */
+
+public class ActionOnClientFacade {
+    private WebDriver driver;
```

Comment:
Why do we need a driver here?

Reply:
because i have methods that return object